modules/table/src/integrationTest/java/org/apache/ignite/distributed/ITTablePersistenceTest.java

```diff
@@ -41,7 +43,7 @@
 import org.apache.ignite.raft.client.service.RaftGroupService;
 
 /**
- * Persistent (rocksdb-based) partitions raft group snapshots tests.
+ * Persistent partitions raft group snapshots tests.
```

Comment:
If this test tests some persistent properties, why do we switch to an in-memory implementation?

Reply:
That's because of poor class name, I think. Nothing fails so I conclude that it didn't test any persistent properties

Reply:
I changed the test a little bit so that storage "survives node restart"